### PR TITLE
Add build target to CNI make file

### PR DESCRIPTION
**Pull request by TerryHowe**
Add build target to CNI make file, so we can change the target for Prow presubmit to what it should use.

### CODE REVIEW

File: projects/containernetworking/plugins/Makefile [hunk 1]
```diff
@@ -22,8 +12,13 @@ binaries:
 tarballs:  binaries
 	build/create_tarballs.sh $(REPO) $(GIT_TAG)
 
+.PHONY: build
+build: tarballs
+	echo "Done $(COMPONENT)"
+
 .PHONY: release
 release: tarballs
+	echo "Done $(COMPONENT)"
```
Comment (vivek-koppuru) [inline projects/containernetworking/plugins/Makefile:21]: So the release target is not uploading to S3 vs build? What is the situation there?

Comment (TerryHowe) [reply projects/containernetworking/plugins/Makefile:21]: That is the change I put in on #137 which hasn't merged yet

Comment (jaxesn) [reply projects/containernetworking/plugins/Makefile:21]: to confirm, you are saying once 137 is merged, the release target will upload after build?  Thats how it looks [here](https://github.com/aws/eks-distro/pull/137/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R66)

**Review by vivek-koppuru — APPROVED** [review]
/lgtm

File: projects/containernetworking/plugins/Makefile (hunk 1 above)
Comment (TerryHowe) [reply projects/containernetworking/plugins/Makefile:21]: This change is just adding a build target to the Make file so the presubmit tests will pass for #137 The #137 PR adds the upload to s3 like you pointed out in your link

**Pull request merged**